atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -108,7 +108,12 @@ const CrashEditCoordsMap = ({
       >
         <FullscreenControl position="top-left" />
         <NavigationControl position="top-left" showCompass={false} />
-        <Marker {...markerCoordinates}>
+        <Marker
+          latitude={markerCoordinates.latitude || defaultInitialState.latitude}
+          longitude={
+            markerCoordinates.longitude || defaultInitialState.longitude
+          }
```

Comment:
Another small fix so that the marker has coordinates when the crash does not. Previously, the marker would only appear after moving the map since it is captured on a map drag event. Now, the initial location of the marker is the same lat/lon used to initialize the map when adding coordinates to a crash that has none.

Reply:
nice touch!